tools/docker/Dockerfile

```diff
@@ -112,17 +112,29 @@ RUN mkdir -p /var/lib/firecracker-containerd/runtime \
 	&& chmod 0444 default-vmlinux.bin \
 	&& mv default-vmlinux.bin /var/lib/firecracker-containerd/runtime/default-vmlinux.bin
 
-COPY --from=firecracker-containerd-build /home/builder/firecracker-containerd /firecracker-containerd
-COPY tools/image-builder/rootfs.img /var/lib/firecracker-containerd/runtime/default-rootfs.img
-COPY --from=firecracker-containerd-build /output/* /usr/local/bin/
-
-RUN --mount=type=cache,from=build-base,source=/home/builder/go/pkg/mod,target=/tmp/go/pkg/mod,readonly \
+RUN --mount=type=cache,from=build-base,source=/home/builder/go/pkg/mod,target=/tmp/go/pkg/mod \
 	mkdir -p ${GOPATH}/pkg/mod \
 	&& cp -R /tmp/go/pkg/mod/* ${GOPATH}/pkg/mod \
 	&& cp -R /tmp/go/pkg/mod/* /home/builder/go/pkg/mod \
 	&& chown -R builder /home/builder/go/pkg/mod
 
+# demo-network installs binaries via "go get" and "go install"
+COPY --from=firecracker-containerd-build \
+    /home/builder/firecracker-containerd/Makefile \
+    /home/builder/firecracker-containerd/go.mod \
+    /home/builder/firecracker-containerd/go.sum \
+    /home/builder/firecracker-containerd/tools \
+    /firecracker-containerd/
+COPY --from=firecracker-containerd-build \
+    /home/builder/firecracker-containerd/tools \
+    /firecracker-containerd/tools
+RUN make -C /firecracker-containerd demo-network
```

Comment:
We are still running demo-network below on line 176, which shouldn't be needed

Reply:
Ah right. I'd like to remove that from firecracker-containerd-integ-test.